src/components/ShareModal/ShareModal.module.scss

```
@@ -0,0 +1,27 @@
+.error-message {
+  margin-top: theme.$spacingSmall;
+}
+
+.visually-hidden {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.9.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)